package.json

```diff
@@ -10,7 +10,7 @@
     "prepare": "husky install",
     "build": "nx run-many --target=build",
     "lint": "nx run-many --target=lint",
-    "test": "nx run-many --target=test -- --watchAll=false --silent=true",
```

Comment:
vitest doesn't have the `watchAll` option, so if you ran this locally, it would incorrectly enter interactive mode in the parallel runner. So let's use a `test:ci` job that is configured for each project.